test/flush_worker_test.cc

```diff
@@ -26,6 +25,8 @@ using ::testing::ReturnRef;
 using ::testing::StrictMock;
 using ::testing::Test;
 
+const int kNumTimerLoops = 100;
```

Comment:
Since this is defined in a fairly large scope, can we add a comment with an explanation of what this is?

Reply:
Done